articles/connections/passwordless/sms-gateway.md

```diff
@@ -1,66 +1,74 @@
 ---
-title: Send One-time Codes via Your Own SMS Gateway
+title: Set Up SMS Gateway for Passwordless Connections
 topics:
     - connections
     - passwordless
     - sms
 contentType: how-to
 useCase: customize-connections
 ---
-# Send One-Time Codes via Your Own SMS Gateway
+# Set Up SMS Gateway for Passwordless Connections
 
-By default, the SMS connection uses Twilio to send the one-time code. If you already have your own infrastructure to send text messages, you cannot or might not want to use Twilio.
+If you are using a Passwordless connection, this guide will show you how you can use custom infrastructure that you've built (instead of Twilio) to send out one-time use codes via SMS messages to users who are logging in.
```

Comment:
Kind of a long sentence. User might get a little lost toward the end. :)

Reply:
Provide a little more context and a link to the how-to that shows them how to set-up SMS Passwordless Connections. (To help provide context and link to the main docs since they could stumble across this page in a search and it would be orphaned.)